docker/build_artifacts/sagemaker/serve.py

```diff
@@ -449,7 +452,7 @@ def start(self):
             self._setup_gunicorn()
             self._start_gunicorn()
             # make sure gunicorn is up
-            with self._timeout(seconds=30):
+            with self._timeout(seconds=self._gunicorn_setup_timeout_seconds):
```

Comment:
You would need to update the `gunicorn_command` as well to consume the timeout. This comment refers to line # 202 in this file where gunicorn command is setup.

Reply:
Good catch. As a result, this PR now includes two env vars. One for the `gunicorn_command` and one for this timeout. This timeout may end up being unnecessary, but it is probably better than just a hardcoded value.

Reply:
Both are effectively the same. The code is using `SIGALARM` to kill the process if the `gunicorn`  process is not up in the specified time (previously hard coded `30` seconds). You should replace `self._gunicorn_setup_timeout_seconds` with `self._gunicorn_timeout_seconds`